access/templates/access/_error_summary.html

```diff
@@ -5,14 +5,17 @@
 	<div class="error-summary__text">
 		<h4>{% trans "Submission failed" %}</h4>
 		<p>{% trans "Check your answers before trying again:" %}</p>
+		<ul>
 		{% for field in result.form %}
 			{% if field.errors %}
-				<p>{{ field.label }}:
-				{% for error in field.errors %}
-					{{ error }}
-				{% endfor %}
-				</p>
+					<li>
+						<strong>{{ field.label }}: </strong>
+						{% for error in field.errors %}
```

Comment:
Now that you are using a html list here, would it make sense to show each field error as a separate list item? Now you have one list item per field, but the field may have multiple errors in the same list item, though it is very rare. What do you think?

Reply:
That crossed my mind, but I kind of like it better, when they are as a list by field, so that the errors of the same field are there one after the other. I am not sure what is the convention here though? That could of course be changed still :)

Reply:
I think either way is fine, so it can be left as is.